5_databases/week2/week2-hmw-part3.sql

```diff
@@ -0,0 +1,33 @@
+SELECT * FROM user;
```

Comment:
according to the homework you should be using this database to solve this requirement no? https://github.com/HackYourFuture-CPH/databases/blob/main/lesson2/lesson2-data.sql

if yes, then your queries won't work because task table has no user_id :smile: if no, can you please point me to the database you used? thank you :pray: